torch/fx/experimental/symbolic_shapes.py

```diff
@@ -59,36 +63,19 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         SYM_FUNCTION_MODE = self.inner
 
 def has_symbolic_sizes_strides(elem):
-    return any([isinstance(i, torch.SymIntNode) for i in elem.shape])
+    return (
+        any([isinstance(i, torch.SymIntNode) for i in elem.shape])
+        or any([isinstance(i, torch.SymIntNode) for i in elem.stride()])
+        or isinstance(elem.numel(), torch.SymIntNode)
+        or isinstance(elem.storage_offset(), torch.SymIntNode)
+    )
```

Comment:
We should optimize this to just query the field on TensorImpl

Reply:
Also, the numel test is unnecessary, if it is symbolic size is symbolic

Reply:
Will remove `numel`, but will do TensorImpl field query in future PR.